src/main/java/net/dries007/holoInventory/Config.java

```diff
@@ -100,7 +99,6 @@ public void doConfig() {
         colorG = configuration.get(HoloInventory.MODID, "colorGreen", colorG, "0-255").getInt();
         colorB = configuration.get(HoloInventory.MODID, "colorBlue", colorB, "0-255").getInt();
         keyMode = configuration.get(HoloInventory.MODID, "keyMode", keyMode, "Valid modes:\n" +
-                "0: Always display hologram.\n" +
```

Comment:
can you leave that in? I think it makes sense to show that this option still exists

Reply:
but it is useless, dont do nothing